src/pages/createOrder.js

```diff
@@ -301,16 +302,21 @@ class CreateOrderPage extends React.Component {
       return false;
     }
 
-    // Check if we're creating a new organization
+    // Check if we're creating a new profile
     if (!this.state.stepProfile.id) {
       this.setState({ submitting: true });
+      if (this.state.stepProfile.name.match(/^anonym/)) {
```

Comment:
Let's say my company is called `anonymous coders` and I pick "Create new organization", won't that mess with the state? Using `name` with a regex doesn't seems very clean, can't we use the `key` instead?

Reply:
How was that resolved?